SampleBase/src/Android/SampleAppAndroid.cpp

```diff
@@ -137,12 +140,16 @@ class SampleAppAndroid final : public SampleApp
                                                         m_SwapChainInitDesc, AndroidWindow,
                                                         &m_pSwapChain);
                 m_TheSample->ResetSwapChain(m_pSwapChain);
-                return m_pSwapChain ? EGL_SUCCESS : EGL_NOT_INITIALIZED;
+                paused = m_pSwapChain;
```

Comment:
Even though c++ allows that, it is not very clear. `store(m_pSwapChain != nullptr)` will make it clearer